runtime/src/lib.rs

```diff
@@ -1328,6 +1328,16 @@ impl_runtime_apis! {
                 vec![]
             }
         }
+
+        fn get_neuron_certificate(netuid: u16, uid: u16) -> Vec<u8> {
+            let _result = SubtensorModule::get_neuron_certificate(netuid, uid);
+            if _result.is_some() {
+                let result = _result.expect("Could not get Certificate");
+                result.encode()
+            } else {
+                vec![]
+            }
```

Comment:
nitpick: I'd personally write it this way. But they seem to write all their methods the round-about way. No need to change this. Just putting it out there.

```suggestion
            SubtensorModule::get_neuron_certificate(netuid, uid).map(|r| r.encode()).unwrap_or(vec![])
```